crawl-ref/source/dat/database/de/wpnnoise.txt

```diff
@@ -1104,71 +1104,71 @@ _SCREAM_
 
 @The_weapon@ @_screams_@, "ICH RIECHE BLUT!"
 
-@The_weapon@ @_screams_@, "ZERSTÖRUNG, NICHT ZAUBEREI!"
+@The_weapon@ @_screams_@, "ZERSTÖRUNG STATT ZAUBERTRICKS!"
 
-@The_weapon@ @_screams_@, "TÖTEN, HASSEN, MUTTIEREN!"
+@The_weapon@ @_screams_@, "TÖTEN, HASSEN, VERSTÜMMELN!"
 
 # Demolition Man
 @The_weapon@ @_screams_@, "MORD! TOD!!! TÖTEN!!!"
 
 @The_weapon@ @_screams_@, "DEZIMIEREN, VERNICHTEN, AUSROTTEN!"
 
-@The_weapon@ @_screams_@, "ICH BADE IN BLUT UND KNOCHEN UND HIRN!"
+@The_weapon@ @_screams_@, "ICH BADE IN BLUT UND KNOCHEN UND GEHIRN!"
 
 # A faint reminiscence to Holst
 @The_weapon@ @_screams_@, "ICH BIN DER BRINGER DES KRIEGES!"
 
-@The_weapon@ @_screams_@, "LASST UNS IHR ELENDES LEBEN BEENDEN!"
+@The_weapon@ @_screams_@, "LASST UNS IHRE ELENDEN LEBEN BEENDEN!"
 
 @The_weapon@ @_screams_@, "BRINGT SIE NIEDER!"
 
 @The_weapon@ @_screams_@, "STIRB, WÄHREND ICH SINGE!"
 
 @The_weapon@ @_screams_@, "KNIETIEF IM BLUT UND IMMER NOCH STARK!"
 
-@The_weapon@ @_screams_@, "DA CAPO! DA CAPO! DA CAPO!"
+@The_weapon@ @_screams_@, "ZUGABE! ZUGABE! ZUGABE!"
 
 @The_weapon@ @_screams_@, "WIE KÖNNT IHR ZU DIESEM ZEITPUNKT AN SINGEN DENKEN?!"
 
-@The_weapon@ @_screams_@, "DIE SCHREIEN! LASST SIE NICHT MIT DEN SCHREIEN AUFHÖREN!"
+@The_weapon@ @_screams_@, "DIE SCHREIE! LASST SIE NICHT MIT DEN SCHREIEN AUFHÖREN!"
 
 @The_weapon@ @_screams_@, "KEINER SOLL[LEBEN|ENTKOMMEN|ÜBERLEBEN|VERSCHONT WERDEN]!"
 
-@The_weapon@ @_screams_@, "SCHONT KEINEN!"
+@The_weapon@ @_screams_@, "VERSCHONT NIEMANDEN!"
 
-@The_weapon@ @_screams_@, "KOMMT ZU MIR! KOMMT ZU MIR UND STIRB!"
+@The_weapon@ @_screams_@, "KOMMT ZU MIR! KOMMT ZU MIR UND STERBT!"
 
 @The_weapon@ @_screams_@, "SCHNEIDEN! SCHNEIDEN! SCHNEIDEN UND SCHLITZEN!"
 
 @The_weapon@ @_screams_@, "STECHEN STECHEN STECHEN!"
 
-@The_weapon@ @_screams_@, "ABANDON ALL HOPE!"
+@The_weapon@ @_screams_@, "GEBT ALLE HOFFNUNG AUF!"
 
-@The_weapon@ @_screams_@, "SCHLITZT SIE! SCHLITZT SIE ALLE!"
+@The_weapon@ @_screams_@, "SCHLITZT SIE AUF! SCHLITZT SIE ALLE AUF!"
```

Comment:
Should this be singular (SCHLITZE)? I think the sword is talking to the player here.